### PR TITLE
Adding context

### PR DESCRIPTION
**Pull request by Pherpher089-zz**
Added context, removed discover, playlist and settings page. 

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: groovin-music/src/components/Home/pages/PlayerPage/PlayerPage.js [hunk 1]
```diff
@@ -2,21 +2,36 @@ import React, { useState } from 'react';
 import './PlayerPage.css';
 
 import SpotifyPlayer from 'react-spotify-player';
-import { gql } from 'apollo-boost';
-import { useQuery, useMutation } from '@apollo/react-hooks';
 
-const CREATE_RECOMMENDED_PLAYLIST = gql`
```
Comment (alemasney) [inline groovin-music/src/components/Home/pages/PlayerPage/PlayerPage.js]: nice work!

File: groovin-music/src/components/Home/pages/PlayerPage/PlayerPage.js [hunk 2]
```diff
@@ -39,91 +54,7 @@ function Player(props) {
...
 }
 
-const RecommendedPlayist = () => {
-  console.log('here in the reced playlist function');
-  const [input, setInput] = useState({
-    name: '',
-    description: '',
-  });
-
-  const [tracks, setTracks] = useState([]);
```
Comment (alemasney) [inline groovin-music/src/components/Home/pages/PlayerPage/PlayerPage.js]: good job using hooks.

**Review by alemasney — APPROVED** [review]
it looks great! good job on getting the player up.

**Review by djzaragoza — APPROVED** [review]
Great work adding context!

**Pull request merged**